api/spec/src/types.tsp

```diff
@@ -171,7 +187,7 @@ scalar CountryCode extends string;
  */
 @friendlyName("Address")
 model Address {
-  country: CountryCode;
+  country?: CountryCode;
```

Comment:
I would make at least the country mandatory. That would help a lot with billing-related decisions. (e.g. Tax country,etc)

Reply:
Country is only required for paid plans / subscriptions. Making it required would be challenging to handle freemium customers where in many cases vendor doesn't have the necessary information.

I think alternatively we should add an interface to BillingProfile that validates if customer is eligible to start a paid subscription. This can be various between providers. For example with manual invoicing you may only need an address and business info while with Stripe you need default payment method too. Some invoicing providers supporting manual action required state may allow temporary missing information etc.

I think subscription module should use BillingProfile to check if condition meet for paid plan. cc @GAlexIHU